dev/regen_file_lists.sh

```diff
@@ -11,6 +11,10 @@ else
     FOLDER=$1
 fi
 
+# force sorting by byte value
+export LC_ALL=C
+
+echo "forcing locale: LC_ALL=$LC_ALL"
```

Comment:
Is it really necessary toe xport it into the calling environment?

Reply:
yah, good point. Don't want to break something else that depends on locales